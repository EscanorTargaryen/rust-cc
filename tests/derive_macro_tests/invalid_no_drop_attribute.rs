use rust_cc::*;

#[derive(Trace, Finalize)]
struct MyStruct {
    #[rust_cc(unsafe_no_drop)]
    a: (),
}

#[derive(Trace, Finalize)]
enum MyEnum1 {
    #[rust_cc(unsafe_no_drop)]
    A(),
}

#[derive(Trace, Finalize)]
enum MyEnum2 {
    A {
        #[rust_cc(unsafe_no_drop)]
        a: (),
    },
}

fn main() {
}
