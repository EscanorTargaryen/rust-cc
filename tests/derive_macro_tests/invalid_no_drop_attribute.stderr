error: Invalid attribute position
 --> tests/derive_macro_tests/invalid_no_drop_attribute.rs:5:15
  |
5 |     #[rust_cc(unsafe_no_drop)]
  |               ^^^^^^^^^^^^^^

error: Invalid attribute position
  --> tests/derive_macro_tests/invalid_no_drop_attribute.rs:11:15
   |
11 |     #[rust_cc(unsafe_no_drop)]
   |               ^^^^^^^^^^^^^^

error: Invalid attribute position
  --> tests/derive_macro_tests/invalid_no_drop_attribute.rs:18:19
   |
18 |         #[rust_cc(unsafe_no_drop)]
   |                   ^^^^^^^^^^^^^^
